test/test_hand_eye.py

```diff
@@ -0,0 +1,36 @@
+def test_init_pose():
+    import numpy as np
+    import zivid.handeye
+
+    elements = [1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1]
+    data = np.array(elements, dtype=np.float64).reshape((4, 4))
+
+    pose = zivid.handeye.Pose(data)
+
+    assert pose is not None
+    assert isinstance(pose, zivid.handeye.Pose)
+
+
+def test_detect_feature_points(point_cloud):
+    import zivid
+
+    feature_points = zivid.handeye.detect_feature_points(point_cloud)
+
+    assert feature_points is not None
```

Comment:
Is it worth noting that this test actually fails at detecting feature-point (since MiscObjects.zdf does not contain a checkerboard). Could be misleading...

Reply:
Perhaps, but simply taking a random point cloud, how likely is it that this pointcloud contains a checkerboard? If it really was a checkerboard in that point cloud, I would have named it `point_cloud_with_checkerboard` or somethin like that